### PR TITLE
fix(FEC-12097): More plugins - Manage the UI & order of the plugin icons

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

solves: FEC-12097 &  FEC-12500, FEC-12497, FEC-12538, FEC-12553

related pr:
https://github.com/kaltura/playkit-js-ui-managers/pull/10

in addition to the required changes from [upper bar manager pr](https://github.com/kaltura/playkit-js-ui-managers/pull/10)
the following general UI bugs were fixed: FEC-12500, FEC-12497, FEC-12538, FEC-12553

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/top-bar/_top-bar.scss [hunk 1]
```diff
@@ -14,7 +14,7 @@
   width: 100%;
   top: 0;
   left: 0;
-  z-index: 1;
+  z-index: 2;
```
Comment (SivanA-Kaltura) [inline src/components/top-bar/_top-bar.scss:17]: why ?

File: src/ui-presets/playback.js [hunk 2]
```diff
@@ -71,6 +71,7 @@ class PlaybackUI extends Component {
             <VideoArea />
             <GuiArea>
               <Fragment>
+                <UnmuteIndication />
```
Comment (SivanA-Kaltura) [inline src/ui-presets/playback.js:74]: why ?

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

File: src/components/top-bar/_top-bar.scss (hunk 1 above)
Comment (JonathanTGold) [reply src/components/top-bar/_top-bar.scss:17]: Cancelled

File: src/ui-presets/playback.js (hunk 2 above)
Comment (JonathanTGold) [reply src/ui-presets/playback.js:74]: Screen Shot 2022-09-11 at 12 50 55

Comment (JonathanTGold) [reply src/ui-presets/playback.js:74]: while top bar contains icons it falls bellow

File: src/styles/_plugin-button.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  cursor: pointer;
+  color: $tone-1;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  opacity: 0.8;
+  border: none;
+  background-color: transparent;
+
+  //&:not(:last-child) {} why the hell doesn't it work in sass ???
```
Comment (SivanA-Kaltura) [inline src/styles/_plugin-button.scss:15]:
```suggestion
```

**Review by SivanA-Kaltura — CHANGES_REQUESTED** [review]

**Review by giladna — APPROVED** [review]

**Review by SivanA-Kaltura — APPROVED** [review]

**Pull request merged**